lib/growl.js

```diff
@@ -23,8 +23,16 @@ var path = require('path');
  * @see {@link Mocha#isGrowlCapable}
  * @return {boolean} whether Growl notification support can be expected
  */
-exports.isCapable = function() {
-  return !process.browser && which(getSupportBinaries()) !== '';
+exports.isCapable = () => {
+  if (!process.browser) {
+    const binaries = getSupportBinaries();
+    if (binaries.length) {
+      try {
+        return Boolean(which(binaries, {all: true}));
```

Comment:
This is broken now... `binaries` is an array, but your `which` takes a string necessitating a `for` loop.
You also only want the first match, so `all` is not as helpful as `nothrow` would have been.